### PR TITLE
Variant.no v2

### PR DESCRIPTION
**Pull request by mikaelbr**
All work on v2 of the new variant.no with updated profile.

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@import "~@variant/profile/colors/index.module.css";
+@import "~@variant/profile/typography/index.module.css";
+
+#root {
+  position: relative;
+}
+
+html {
+  background: var(--color-secondary4__tint2);
```
Comment (adriansberg) [inline src/app.css:21]: Any reason to separate this rule from the other rule at the top of the file?

Comment (mikaelbr) [reply src/app.css:21]: Not really, but just separating the "global general configuration" from the project specific. And in order for some specificity from the profile/color included above.

Comment (adriansberg) [reply src/app.css:21]: I see 👌

**Comment by mikaelbr** [comment]
@MathiasFl I think actually `AZURE_PROXY_URL` and `AZURE_PROXY_KEY` should be public now that we limit the data. It's essentially just a JSON representation of what we already have. And it makes it sooo much easier to test it out locally.

**Comment by mikaelbr** [comment]
This is merged now! Good job everyone.

**Pull request closed without merging**